### ISSUE TITLE
(ts-plugin) False-positive error.  Something about jsx

### ISSUE THREAD

**Issue by vegerot** [znck/vue-developer-experience#46]
I don't use jsx or have any references to jsx in my project.  In a valid `template` tag I get this error:

```vue
<template>
  <v-autocomplete
    :hide-details="hideDetails"
    type="text"
    filled
    background-color="stone"
    :value="value"
    :items="items"
    :multiple="multiple"
    v-bind="$attrs"
    :menu-props="{ auto: true, ...menuProps }"
    class="styled-autocomplete"
    v-on="$listeners"
    @input="$emit('input', $event)"
    @change="$emit('change', $event)"
    @blur="$emit('blur', $event)"
  >
    <template
      v-if="multiple && selectAll && !$scopedSlots['prepend-item']"
      #prepend-item
    >
      <v-list-item
        ripple
        @click="toggleAllSelected"
      >
        <v-list-item-action>
          <v-icon :color="items.length > 0 ? 'indigo' : 'unset'">
            {{ selectedIcon }}
          </v-icon>
        </v-list-item-action>
        <v-list-item-content>
          <v-list-item-title>
            Select All
          </v-list-item-title>
        </v-list-item-content>
      </v-list-item>
    </template>
    <template
      v-for="(_, slot) of $scopedSlots"
      #[slot]="scope"
    >
      <slot
        :name="slot"
        v-bind="scope"
      />
    </template>
  </v-autocomplete>
</template>
```
<img width="1011" alt="image" src="https://user-images.githubusercontent.com/5297725/98168985-72477200-1eb9-11eb-8090-c71e4be4b41c.png">

As you can see, the error starts in the middle of the class name and extends to nearly the start of the closing tag.

Don't know if it's related, but when I hover some listeners, I get seemingly incorrect signature lookup

<img width="877" alt="image" src="https://user-images.githubusercontent.com/5297725/98169110-aa4eb500-1eb9-11eb-9f76-7114f96f9045.png">

**Comment by znck** [comment]
Related to #48

**Comment by znck** [comment]
Multiple fixes in #78, #99, #105; It's not perfect but much better. Closing this and will continue tracking #48.
